exporter/logzioexporter/testdata/config.yaml

```diff
@@ -0,0 +1,17 @@
+receivers:
+  examplereceiver:
+
+processors:
+  exampleprocessor:
+
+exporters:
+  logzio:
```

Comment:
I believe the preferred practice is to have multiple component instances, exercising defaults and configuring all possible fields: https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/jaegerexporter/testdata/config.yaml#L7